pages/index.js

```diff
@@ -1,22 +1,30 @@
-import Head from "next/head";
 import styled from "styled-components";
+import DateDisplay from "../components/DateDisplay";
+import Task from "../components/Task";
+import Welcome from "../components/Welcome";
 
 export default function Home() {
   return (
-    <div>
-      <Head>
-        <title>My App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <Main>
-        <h1>My App</h1>
-      </Main>
-    </div>
+    <Wrap>
+      <WrapMask>
+        <Welcome />
+        <DateDisplay />
+        <Task />
+      </WrapMask>
+    </Wrap>
   );
 }
 
-const Main = styled.main`
-  text-align: center;
+const Wrap = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background: url(/images/bgImage.png);
+  backdrop-filter: blur(2px);
+`;
+
+const WrapMask = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background: rgba(253, 231, 190, 0.4);
+  backdrop-filter: blur(40px);
 `;
```

Comment:
What happens if the content becomes longer than your background? This seems as if it might cause problems later on.

Reply:
Did you make any changes here?

Reply:
@Roland-Hufnagel yes i have changed my code in `Files changed`, this one is outdated